### DISCUSSION TITLE
LinkDialog with currentSelection not working with react-dom 18.3.1 or next-js 14.1.0

### DISCUSSION THREAD

**Discussion by dedalusgranger** [mdx-editor/editor#525] [Q&A]
Hi, I'm trying to use useCellValue(currentSelection$). It works when using react-dom 18.2.0.
But I get an error when using react-dom 18.3.1 and next-js 14.1.0. Is mdx-editor not compatible with these?

Error: Unable to find an active editor state. State helpers or node methods can only be used synchronously during the callback of editor.update() or editorState.read().

attached differing call stacks for react-dom.
Working react-dom 18.2.0:
![react-dom 18 2 0 stack](https://github.com/mdx-editor/editor/assets/174740221/d054b52c-29d5-4210-80b2-5b0caa4de128)
NOT working react-dom 18.3.1:
![react-dom 18 3 1 stack](https://github.com/mdx-editor/editor/assets/174740221/47e8514e-55dc-40f6-ab1b-d136d050164b)

**Comment by petyosi** [comment]
No idea, haven't tested it yet. Ensure that you don't have multiple React and Lexical versions.
